src/Reporter.hpp

```diff
@@ -196,6 +196,7 @@ namespace geopm
             double m_total_time;
             double m_overhead_time;
             double m_sample_delay;
+            std::string m_profile_name;
```

Comment:
```suggestion
            const std::string m_profile_name;
```
I think this can be const.  I only see it being set in the initializer list.